src/Assessor/AssessorRequestExporter.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace AppBundle\Assessor;
+
+use AppBundle\Normalizer\AssessorRequestExportNormalizer;
+use AppBundle\Serializer\XlsxEncoder;
+use Symfony\Component\Serializer\Serializer;
+
+class AssessorRequestExporter
+{
+    public const DEFAULT_FILE_NAME = 'bureaux-de-vote-demandes-assesseurs.'.XlsxEncoder::FORMAT;
+
+    /** @var AssessorRequestExportNormalizer */
```

Comment:
docblock not needed since the typehint of this property is made by the constructor

Reply:
I writed this dockblock to generate the constructor with my IDE, I forgot to remove it, thanks ! 
Nice catch :+1: